infrastructure/templates/public-api/components/functionApp.bicep

```diff
@@ -68,67 +95,63 @@ resource appServicePlan 'Microsoft.Web/serverfarms@2023-01-01' = {
   }
 }
 
-var dedicatedStorageAccountName = replace('${subscription}eessa${functionAppName}', '-', '')
+// Configuring a single shared storage account for task management, and 2 individual storage accounts to be split
+// between the production slot and staging slot. See
+// https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-zero-downtime-deployment#status-check-with-slot
+var durableManagementStorageAccountName = replace('${subscription}eessa${functionAppName}mg', '-', '')
+var slot1StorageAccountName = replace('${subscription}eessa${functionAppName}s1', '-', '')
+var slot2StorageAccountName = replace('${subscription}eessa${functionAppName}s2', '-', '')
```

Comment:
The `DurableManagementStorage` connection string are set using the outputs of the `slot1StorageAccountModule` and `slot2StorageAccountModule` which use the names of `slot1StorageAccountName` and `slot1StorageAccountName`.

I think the `AzureWebJobsStorage` connection string is for the shared storage account, and that its name should be something like `sharedStorageAccountName` here?